perception/bytetrack/include/bytetrack/bytetrack.hpp

```diff
@@ -0,0 +1,62 @@
+// Copyright 2022 Tier IV, Inc.
```

Comment:
```suggestion
// Copyright 2023 TIER IV, Inc.
```

Reply:
Because the files under `lib` directory have been slightly modified to follow the autoware coding guideline (also, some of them are renamed to be lower-snake-case),  I guess TIER IV's license statements might be allowed to be there. If you think the statements should not be there, I'll remove them, so could you let me know your thought?

Reply:
Thank you for your reply.
IMO, 
- Only changing lower-snake-case is difficult to claim as a copyrighted work.
- Also, the original license is not written in the file. It looks like a TIER IV work.

So, If you want to claim it as a copyrighted work, it it better to write it under the original license.

Reply:
@yukkysaito 
I appreciate your instruction. By adding copyright statements, I just would like to indicate these codes are not pure copies and need attention to read them. I agree with your opinion, so I  re-arranged the order of the copyright statements (55b00a10196f75457f3c40b3b844536319174e19)